### DISCUSSION TITLE
May i join your team

### DISCUSSION THREAD

**Discussion by VitorGSC** [ROM-builders/temporary#11478] [General]
Builds successfully:
 - https://cirrus-ci.com/task/6128340972077056
 - https://cirrus-ci.com/build/5324610521530368
 - https://cirrus-ci.com/build/4879474222170112

Device Specific DT: https://github.com/VitorGSC/device_motorola_ali
Device Common: https://github.com/VitorGSC/device_motorola_msm8953-common
Vendor DT: https://github.com/VitorGSC/vendor_motorola_ali
Vendor Common: https://github.com/VitorGSC/vendor_motorola_msm8953-common
Kernel: https://github.com/VitorGSC/kernel_motorola_msm8953

Telegram ID: @Vitor_GSC

**Comment by Apon77** [comment]
please follow #learn from telegram group
